### DISCUSSION TITLE
Welcome to Generative AI For Beginners Discussions!

### DISCUSSION THREAD

**Discussion by jimbobbennett** [microsoft/generative-ai-for-beginners#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ovanjohn** [comment]
Am a beginner, how do I get started?

**Reply by vanderghast** [reply]
Indeed, good, very good question. I finally found that we could start at about four pages down at "Course Introduction". Ok, strange place to place the first handshake. But next, an hyperlink to fork the entire repo.  I clicked on the hyperlink, the fork is created.  And now, what's up doc? I have to create a "code space"?  What does it eat in winter, a code space?  I am not even in Visual Studio. I should have? Ok. Maybe. Open VS, and now, I am lost in the nowhere void. So, I come back, Click the hyperlink to create the fork, Error, claims about it is already created. Where, on Mars? 
Is there really a "Start Here" button?

**Comment by Ovanjohn** [comment]
My name is Nnadozie Obi,
With the username Ovanjohn, am a Civil Engineer, just started learning python. Am a beginner and want to go full time into programming

**Reply by MorrisMuuoMulitu** [reply]
Excellent! Here's to a journey filled with insightful learning. 😊

**Reply by frankiedre** [reply]
> My name is Nnadozie Obi,
> With the username Ovanjohn, am a Civil Engineer, just started learning python. Am a beginner and want to go full time into programming

**Comment by Anurag9901** [comment]
Hello! My name is Anurag Topno. I am a btech Computer science engineering student.

**Reply by L-uz** [reply]
From where u r completing your degree?

**Comment by drewhart64** [comment]
Hi I'm David. I'm new here I really find good interest using this gifthub for so many things but I need to learn how to do it. Is anyone here that teaches ?

**Reply by Kill0geR** [reply]
I can try and teach you. What do you want to learn specifically

**Comment by sam951013** [comment]
Can I translate to Chinese Traditional(ZH-TW)?

**Comment by vanderghast** [comment]
Hi!

First, I am NOT from Microsoft.

That being said, a company name is technically required, today, to accept check of money tranfer sent to the said name. If you don't plan to receive such payment (and be happy with credit to your account, as example, if that ever occur), then you could use your OWN name and your own email. You will be billed under your name, though, if that ever happen that you have to pay for some options  (such as exceeding the an amount of data transfer or require a better bandwidth, etc.)

I am not a lawyer too, neither a fiscalist, but that is what is generally acceepted as interpretation of what is a company, in Canada at least. Note: do not use the "inc." abbreviaton for incorporated if you are not an officially registred  corporation (federal or provincial).
________________________________
De : Hexagonal-Columbus ***@***.***>
Envoyé : 26 novembre 2023 06:24
À : microsoft/generative-ai-for-beginners ***@***.***>
Cc : vanderghast ***@***.***>; Comment ***@***.***>
Objet : Re: [microsoft/generative-ai-for-beginners] Welcome to Generative AI For Beginners Discussions! (Discussion #1)


To apply Azure OpenAI services I need to provide my company email address and company name and address... So do I need to start my own company to take this course?

**Comment by sam951013** [comment]
Hi! I am a students in Taiwan.

**Reply by Karload** [reply]
[]()

**Comment by Tswifty1998** [comment]


**Comment by drewhart64** [comment]
Hi Microsoft/Generative-Ai-For-Beginners

**Comment by LewisBryanM** [comment]
Hi guys new to Azure AI can't wait to see how the gen AI offering develops for businesses and how these can be deployed.

**Comment by JosefernandezG** [comment]
Buenos Días, con todos soy nuevo en esta plataforma y quisiera saber si al termino de seguir el curso de Inteligencia artificial (IA) con el plan de estudios de 24 lecciones y 12 semanas de Microsoft. podre obtener un certificado, es por motivos de por registrarlo en mi Institución.

**Comment by TimothyMichaelGould** [comment]
I have used Ai for a few of my projects. Still has a long way to go

**Comment by Kill0geR** [comment]
Hi guys I am trying to program an ai that writes job-applications in german.

**Comment by ghost** [comment]
Dear members, I’m currently doing research on the subject of using Generative A.I. as a digital designer. What our research group would like to know is which ethical issues have a big impact on the decisions you guys and girls make using generative A.I.
Whether you’re using A.I. or not, we would really like to know your vision and opinion about this subject. Please empty your thoughts and oppinion into your answers, we would like to get as much information as possible.
Are you currently using A.I. when doing your job? Yes, what for. No (not yet), why not?
Using A.I., would you use real information or alter names/numbers to get an answer?
What information would or wouldn’t you use? If the client is asking/ordering you to do certain things that go against your principles, would you still do it because order is order? How far would you go?
Who is responsible for the outcome of the generated content, you or the client?
Would you still feel like a product owner if it was co-developed with A.I.?

What we are looking for is that we would like to know why people do or don’t use AI in the field of design and wich ethical considerations they make. We’re just looking for general moral line of people, for example: 70% of designers don’t feel owner of a design that is generated by AI but 95% feels owner when it is co-created.
So therefore the questions we asked, we want to know the how you feel about this.

**Reply by Himavigithub** [reply]
Yaa for studying purpose we use AI it is good we get quick answers but it will not be perfect answers everytime sometimes it may give different answers which is not usefull

**Comment by Himavigithub** [comment]
I am Himavi studying BE 
Really interested real world project coding etc in many things so joined github

**Comment by EmolOrbit** [comment]
how do we combine robotics with ai

**Comment by Cr4N31** [comment]
Hi I just started JavaScript after learning html and css

**Comment by Bhavish089** [comment]
I never build an AI yet or worked on API like basically there are very less sites which tells a proper procedure how to do it I am looking forward to this respository that I will learn something new

**Comment by albineb007** [comment]
I Guess it will be Useful and Informative!!

**Comment by BrightMontain** [comment]
new AI, ty for contribute, coding windows from scratch for 3 days https://github.com/BrightMontain/ByteBridgeCore/blob/main/README.md

**Comment by OmarElshafei-oss** [comment]
Hello 
My name is omar 
I’m an mechatronics engineer 
I’m  very obssesed how to learn making robots 
And how to using AI in this field 
I have many projects in robots 
And I want to learn more about AI

**Comment by robmdz** [comment]
Hi, my name is Robinson, I'm an economics student with an interest in data science and AI. 

My favorite languages are Python and JavaScript, maybe next month I will learn Go.

**Comment by abhishekprajapati532006** [comment]
Excellent 👍

**Comment by Zoecartercomgmail** [comment]
Hey, I was wondering if anyone could help me find someone to create a fully AI agent running drop shipping business

**Comment by shahinmullan** [comment]
What is generative ai?

**Comment by vanderghast** [comment]
Roughly, it can be seen as an AI which "generates" new content:  text, picture, music, ...  By opposition to AI which extracts "law" or "behavior" from data, or which uses a model to predict from a sequence, as example. Note that the boundary is not always a clean one, such as "generating" an answer to a question can be seen as predicting the next word(s) of a sequence:   "What is the capital of the Malawi?"  is like predicting the next word following: "The capital of the Malawi is ... ".
________________________________
De : shahinmullan ***@***.***>
Envoyé : 2 juillet 2025 03:53
À : microsoft/generative-ai-for-beginners ***@***.***>
Cc : vanderghast ***@***.***>; Comment ***@***.***>
Objet : Re: [microsoft/generative-ai-for-beginners] Welcome to Generative AI For Beginners Discussions! (Discussion #1)


What is generative ai?

**Comment by Emmanuel-Oyewole** [comment]
Hi, I'm Emmanuel, A computer science student from Nigeria, I'm currently working on a project that leverages RAG, got stuck along the way, I'm using Llamaindex frame work, Hoping to get help here
